Umweltzone/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="action_cities">Villes</string>
+    <string name="action_share_app">Partagez cette application&#8230;</string>
+    <string name="action_changelog">Informations sur la version</string>
+    <string name="share_intent_message">
+        "<![CDATA[Essayez Umweltzone - l&#8217;application Android qui a toutes les informations
+        sur les zones à faible pollution et les zones d'interdiction des diesels en Allemagne.
+        \nTéléchargez-la ici gratuitement : https://play.google.com/store/apps/details?id=de.avpptr.umweltzone]]>"</string>
+    <string name="map_my_location_activation_hint">
+        "Cliquez ici pour configurer votre location et afficher votre position sur la carte."</string>
+    <string name="feedback_activity_title">Retours</string>
+    <string name="appinfo_application_logo_description">"Logo de l&#8217;application &#8220;Umweltzone&#8221;"</string>
+    <string name="about_activity_title">A propos</string>
+    <string name="appinfo_subtitle">
+        Une application Android à propos des zones à faible émissions (Umweltzonen en allemand) et
+        des zones d&#8217;interdiction du diesel en Allemagne.</string>
+    <string name="appinfo_copyright_title">Idée et réalisation</string>
+    <string name="appinfo_copyright_content">"         (&#169;) Tobias Preuss, 2013 &#8211; 2020     "</string>
```

Comment:
Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.